docs/UsersGuide/source/LAMGrids.rst

```diff
@@ -1,115 +1,155 @@
 .. _LAMGrids:
 
-========================================================================
-Limited Area Model (LAM) Grids:  Predefined and User-Generated Options
-========================================================================
-In order to set up the workflow and experiment generation of the UFS SRW App, the user
-must choose between three predefined FV3-LAM grids or generate a user-defined grid.
-At this time, full support will only be provided to those using one of the three predefined
+=================================================================================
+Limited Area Model (:term:`LAM`) Grids:  Predefined and User-Generated Options
+=================================================================================
+In order to set up the workflow and generate an experiment with the UFS SRW App, the user
+must choose between four predefined :term:`FV3`-:term:`LAM` grids or generate a user-defined grid.
+At this time, full support will only be provided to those using one of the four predefined
 grids supported in this release. However, preliminary information is provided at the end of
 this chapter that describes how users can leverage the SRW App workflow scripts to generate
-their own user-defined grid. This feature is not fully supported at this time and is
-"use at your own risk".
+their own user-defined grid. Currently, this feature is not fully supported and is
+"use at your own risk."
 
 Predefined Grids
-================
-The UFS SRW App release includes three predefined LAM grids that users can choose from
-prior to generating a workflow/experiment configuration.  To select a predefined grid,
+=================
+The UFS SRW App release includes four predefined limited area model (:term:`LAM`) grids to choose between
+prior to configuring an experiment and generating a workflow. To select a predefined grid,
 the ``PREDEF_GRID_NAME`` variable within the ``config.sh`` script needs to be set to one
-of the following three options:
+of the following four options:
 
 * ``RRFS_CONUS_3km``
+* ``SUBCONUS_Ind_3km``
 * ``RRFS_CONUS_13km``
 * ``RRFS_CONUS_25km``
 
+These four options are provided for flexibility related to compute resources and supported physics options. The high-resolution 3-km :term:`CONUS` grid generally requires more compute power and works well with three of the four supported physics suites (see :numref:`Table %s <GridPhysicsCombos>`). Low-resolution grids (i.e., 13-km and 25-km domains) require less compute power and should generally be used with the fourth supported physics suite: ``FV3_GFS_v16``. 
+
+..
+   COMMENT: FV3_WoFS can run on a 13-km and 25-km grid according to Yunheng. Can the HRRR and RRFS_v1beta also run on the 13-km and 25-km grids? Why is FV3_GFS_v16 preferred? Because it has cumulus physics?
+
```

Comment:
The HRRR and RRFS_v1beta SDFs can run on 13- and 25-km grids, but they should not be.  There is not cumulus/convective parameterization in those SDFs, which is required at those scales.  FV3_GFS_v16 is preferred because it includes that kind of parameterization.